doc/changelog.rst

```diff
@@ -116,10 +116,17 @@ Breaking Changes in 4.0
   :class:`~bson.dbref.DBRef`.
 - The "tls" install extra is no longer necessary or supported and will be
   ignored by pip.
+- ``directConnection`` URI option and keyword argument to :class:`~pymongo
+.mongo_client.MongoClient` defaults to ``False`` instead of ``None``,
```

Comment:
The `pymongo.mongo_client.MongoClient` has to be on one line or else it will not render correctly. See the rendered view here: https://pymongo--716.org.readthedocs.build/en/716/changelog.html

Reply:
Done.